packages/calypso-color-schemes/src/shared/color-schemes/_default.scss

```diff
@@ -252,11 +252,11 @@
 	--color-jetpack-plan-premium: var( --studio-jetpack-green-30 );
 	--color-jetpack-plan-professional: var( --studio-purple-30 );
 
-	--color-masterbar-background: var( --studio-blue-60 );
-	--color-masterbar-border: var( --studio-blue-70 );
+	--color-masterbar-background: var( --studio-gray-100 );
+	--color-masterbar-border: #333;
 	--color-masterbar-text: var( --studio-white );
-	--color-masterbar-item-hover-background: var( --studio-blue-70 );
-	--color-masterbar-item-active-background: var( --studio-blue-90 );
+	--color-masterbar-item-hover-background: #333;
+	--color-masterbar-item-active-background: #23282d;
```

Comment:
@sfougnier @lcollette 
These two colors (#333, #23282d) are the exact values from wp-admin. If I understand the new Navigation Unification, Design Update (see p2-pbAPfg-13V) correctly, it seems like for brand consistency we're starting to tweak wp-admin colors a bit too. If so could you let me if these should be replaced with color studio vars and if so which ones? Thank you!